executor/oci/resolvconf.go

```diff
@@ -16,7 +16,7 @@ var notFirstRun bool
 var lastNotEmpty bool
 
 // overridden by tests
-var resolvconfGet = resolvconf.Get
+var resolvconfPath = resolvconf.Path
```

Comment:
FWIW, depending on what kind of network we're using, this Path may (or may not) be correct; does BuildKit use custom docker networks? (as in that case, if should likely use systemd resolvd's address, not the external DNS it's forwarding to) 🤔

Reply:
^^ let me add that this doesn't change in this PR, just an observation and something we should check.